scripts/wait_for_mongo.py

```diff
@@ -58,3 +74,9 @@ def mongo_hello_wait(mongo_client=mongo_client, retries=10, timeout_ms=10000):
         retries = int(sys.argv[1])
 
     mongo_hello_wait(retries=retries)
+    try:
+        create_user()
```

Comment:
@Alopalao, since we're always dropping the database to ensure a clean slate on every time, I think we could get rid of this initial initial user creation here, even though initially it was helpful. If a developer also sets wrong environment variables when running the tests, the `TestE2EKytosAuth` test suite will fail accordingly, so that's great.

Reply:
Makes sense. Got it.